update chopin posts (#37)
apiobuild/apiobuild.github.io: content/blog/create-an-online-store-for-free.md

It’s hard to decide which approach suits you best, and there are many pros and cons. Why waste time on researching which platform to sell when you could've started selling your product already!

**[Chopin by apio](https://telescope.apiobuild.com/app/chopin) is a lightweight static online store generator.** After providing store information and two google sheets (catalog/order), Chopin will generate a static website with a unique url.

<img src="/images/blog/chopin-architecture.png"  class="banner-img">

#### Demo

In the [demo store](https://trampoline.apiobuild.com/router/chopin/store/page/google-oauth2%7C117090713962028193035/7a8c0376-0fd0-4093-894f-e6d0200444d4), we've configured one google sheet to be catalog and another one to store order.

> You can easily update products to the catalog on google sheet and the store will update in real-time. On the other side, when user submits an order, you will immediately see google sheet updated with order details. [Check out the store yourself now](https://trampoline.apiobuild.com/router/chopin/store/page/google-oauth2%7C117090713962028193035/7a8c0376-0fd0-4093-894f-e6d0200444d4).

Reviewer: ```suggestion
You can easily update products to the catalog on google sheet and the store will update in real-time. When user submits an order, you will immediately see google sheet updated with order details. [Check out the store yourself now](https://trampoline.apiobuild.com/router/chopin/store/page/google-oauth2%7C117090713962028193035/7a8c0376-0fd0-4093-894f-e6d0200444d4).
```

<video width="100%" loop="true" autoplay="true" controls style="align: center">
<source src="/video/gsheet-to-store.mp4" type="video/mp4" />
</video>

## Benefits

#### Google Sheet Backend

> Chopin made creating your own store as easy as filling out a simple survey. You only need to provide your logo and store information as well as google sheets with the catalog and order data respectively, then your shopping site will be generated.

Reviewer: ```suggestion
Chopin made creating your own store as easy as filling out a simple survey. You only need to provide your logo, simple information about your store, two google sheets: one with the catalog and one with order data. That's, it, your web store is ready to go!
```

> **No need to code**, just use the interface everyone’s familiar with - Google Sheet. You can add new items by simply pasting text in a google sheet. Have you ever thought updating a website can be this easy?

Reviewer: ```suggestion
**No code required**, just use the interface everyone’s familiar- Google Sheet. You can add new items by simply by adding a new row in your google sheet. Have you ever thought updating a website can be so easy?
```

**All Chopin generated sites are SSL certified and meet the highest security standard**, which means your customers’ sensitive data (email addresses, phone number, names etc.) are encrypted and protected from unauthorized users. This feature doesn’t just protect store owners and their customers, is also the required element for implementing third-party payment gateways (i.e. [Stripe](https://stripe.com/), [Square](https://squareup.com/), [PayPal](https://www.paypal.com/)).

## Features

#### Customization 

> The website is fully customizable. You can update **logo, store description, background color, and background image** within the [Chopin's configure tab](https://telescope.apiobuild.com/app/chopin/configure) to create a store aligned with your brand. More customization options are also available though simple HTML blocks.

Reviewer: ```suggestion
The website is fully customizable. You can provide **logo, store description, background color, and background image** on [Chopin's configure tab](https://telescope.apiobuild.com/app/chopin/configure) to create a store page that aligns with your brand. More customization options are also available through simple HTML blocks.
```

<img src="/images/blog/store.png" class="post-img"> 

#### Communication 

> Chopin enables social plug-ins so your customers can easily contact you via **Email, Instagram, Facebook, and Line**. You also have the option to setup **announcement** to broadcast important updates or discount information. 

Reviewer: ```suggestion
Chopin enables social plug-ins so your customers can easily reach you via **Email, Instagram, Facebook, and Line**. You also have the option to add **announcement** to broadcast important updates or discount information. 
```

<img src="/images/blog/social.png" class="post-img">

#### Category

> You have the option to group your products in various categories or collections, and customers can go to a certain category directly from the menu. You can even setup a "Featured" collection to highlight certain items.

Reviewer: ```suggestion
You can group products in various categories, and customers can filter products by category directly from the menu. We also support multiple categories per product. For example, you can set up a "Featured" collection to highlight popular items on the top and still show it in other categories so customers to locate any items easily. 
```

<video width="75%" loop="true" autoplay="true" controls style="align: center">
<source src="/video/jump-to.mp4" type="video/mp4" />
</video>

#### Limit Per Customer/Out of Stock

Set a maximum quantity (per customer) for products in the catalog google sheet. Once the customer’s selection reaches the maximum, the system will show the limitation or out of stock message.

<video width="75%" loop="true" autoplay="true" controls style="align: center">
<source src="/video/out-of-stock.mp4" type="video/mp4" />
</video>

#### Tax and Shipping Options

> Depending on where your business is registered, tax rate can be set by you. You can also setup shipping rules, including different destination with different amounts and free shipping minimum.

Reviewer: ```suggestion
Depending on where your business is registered, the tax rate can be set by you. You can also add shipping rules, including different destinations with different amounts and a free shipping minimum.
```
